code/modules/mob/dead/new_player/sprite_accessories/wings.dm

```diff
@@ -27,6 +27,90 @@
 	dimension_y = 34
 	locked = TRUE
 
+// Decorative wings
+/datum/sprite_accessory/deco_wings
+	icon = 'icons/mob/wings.dmi'
+	
+/datum/sprite_accessory/deco_wings/plain
+	name = "Plain"
+	icon_state = "plain"
+
+/datum/sprite_accessory/deco_wings/monarch
+	name = "Monarch"
+	icon_state = "monarch"
+
+/datum/sprite_accessory/deco_wings/luna
+	name = "Luna"
+	icon_state = "luna"
+
+/datum/sprite_accessory/deco_wings/atlas
+	name = "Atlas"
+	icon_state = "atlas"
+
+/datum/sprite_accessory/deco_wings/reddish
+	name = "Reddish"
+	icon_state = "redish"
+
+/datum/sprite_accessory/deco_wings/royal
+	name = "Royal"
+	icon_state = "royal"
+
+/datum/sprite_accessory/deco_wings/gothic
+	name = "Gothic"
+	icon_state = "gothic"
+
+/datum/sprite_accessory/deco_wings/lovers
+	name = "Lovers"
+	icon_state = "lovers"
+
+/datum/sprite_accessory/deco_wings/whitefly
+	name = "White Fly"
+	icon_state = "whitefly"
+
+/datum/sprite_accessory/deco_wings/punished
+	name = "Burnt Off"
+	icon_state = "punished"
+	locked = TRUE
+
+/datum/sprite_accessory/deco_wings/firewatch
+	name = "Firewatch"
+	icon_state = "firewatch"
+
+/datum/sprite_accessory/deco_wings/deathhead
+	name = "Deathshead"
+	icon_state = "deathhead"
+
+/datum/sprite_accessory/deco_wings/poison
+	name = "Poison"
+	icon_state = "poison"
+
+/datum/sprite_accessory/deco_wings/ragged
+	name = "Ragged"
+	icon_state = "ragged"
+
+/datum/sprite_accessory/deco_wings/moonfly
+	name = "Moon Fly"
+	icon_state = "moonfly"
+
+/datum/sprite_accessory/deco_wings/snow
+	name = "Snow"
+	icon_state = "snow"
+
+
+/datum/sprite_accessory/deco_wings/angel
+	name = "Angel"
+	icon_state = "angel"
+	color_src = 0
+	dimension_x = 46
+	center = TRUE
+	dimension_y = 34
+	locked = TRUE
```

Comment:
btw, this is why you don't see them in the list

Reply:
ok so I'll remove that